internal/services/loadbalancer/lb_resource.go

```diff
@@ -56,9 +56,15 @@ func resourceArmLoadBalancer() *pluginsdk.Resource {
 		CustomizeDiff: pluginsdk.CustomDiffWithAll(
 			pluginsdk.ForceNewIf("frontend_ip_configuration", func(ctx context.Context, d *schema.ResourceDiff, meta interface{}) bool {
 				old, new := d.GetChange("frontend_ip_configuration")
-				if len(old.([]interface{})) == 0 && len(new.([]interface{})) > 0 || len(old.([]interface{})) > 0 && len(new.([]interface{})) == 0 {
+				switch {
+				case len(old.([]interface{})) == 0 && len(new.([]interface{})) > 0:
 					return false
-				} else {
+				case len(old.([]interface{})) > 0 && len(new.([]interface{})) == 0:
+					// Azure does not allow an LB to have all frontend removed, it results in the following error:
+					// Error: "Deleting all frontendIPConfigs from load balancer is not supported."
+					// If the old config had frontends, and new config has none, need to force new LB
+					return true
```

Comment:
This should be reflected in the docs to let users know that the resource will be recreated if they remove the block. Can you add a Note?